sdks/core/src/main/java/org/gbif/pipelines/core/parsers/VocabularyParser.java

```diff
@@ -59,6 +59,7 @@
   }
 
   /** @return a life stage parser. */
+  @Deprecated
   public static VocabularyParser<LifeStage> lifeStageParser() {
     return new VocabularyParser<>(LST_PARSER, DwcTerm.lifeStage);
   }
```

Comment:
As I understand we don't use it at all, maybe just delete?

Reply:
right, I just deleted it